<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mPlotViewStyleDialog</class>
 <widget class="QDialog" name="mPlotViewStyleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>曲线样式设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>标题</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>428</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
         <item>
          <widget class="QPushButton" name="pushButton_12">
           <property name="text">
            <string>标题文字</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_8"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>170</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>坐标轴</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>常数坐标</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>对数坐标</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>坐标类型</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>基数</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBox_2">
           <item>
            <property name="text">
             <string>X轴</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Y轴</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>轴</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_5"/>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>范围</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_11">
           <property name="text">
            <string>轴标题</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_7"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page"/>
         <widget class="QWidget" name="page_2">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>刻度线生成方式</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_3">
               <item>
                <property name="text">
                 <string>自动生成</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>等差步长</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>等比步长</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>自定义</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QStackedWidget" name="stackedWidget_2">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page_5"/>
             <widget class="QWidget" name="page_4">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="lineEdit_2"/>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="lineEdit"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QPushButton" name="pushButton_7">
                   <property name="text">
                    <string>中刻度步数</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QPushButton" name="pushButton_6">
                   <property name="text">
                    <string>主刻度步长</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0" colspan="2">
                  <spacer name="verticalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_3">
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="1" column="0">
                  <widget class="QPushButton" name="pushButton_10">
                   <property name="text">
                    <string>中刻度步数列表</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="lineEdit_6">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="placeholderText">
                    <string>请输入每两个主刻度之间的中刻度步数,用逗号隔开</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QPushButton" name="pushButton_9">
                   <property name="text">
                    <string>主刻度列表</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="lineEdit_3">
                   <property name="statusTip">
                    <string>请输入需要生成的主刻度值,用逗号隔开</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                   <property name="placeholderText">
                    <string>请输入需要生成的主刻度值,用逗号隔开</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <spacer name="verticalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_8">
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>栅格</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>栅格显示</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>曲线名称</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,3">
         <item>
          <widget class="QComboBox" name="comboBox_4"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_9"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>155</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
